### ISSUE TITLE
[Bug]: S220 does not work with Homebridge plugin.  shows up as switch

### ISSUE THREAD

**Issue by Infallibleman** [homebridge-eufy-security/plugin#492]
### What happened?

[2023-12-28, 7:05:33 a.m.] Homebridge v1.7.0 (HAP v0.11.1) (HB Hasen) is running on port 51259.
[12/28/2023, 07:05:39]  [EufySecurity]  INFO    Updating existing accessory: Cam-2
[12/28/2023, 07:05:39]  [EufySecurity]  INFO    Updating existing accessory: Cam-1
[12/28/2023, 07:05:41]  [EufySecurity]  WARN    **This accessory is not compatible with HomeBridge Eufy Security plugin:** Cam-2 Type: 63
[12/28/2023, 07:05:41]  [EufySecurity]  WARN    This accessory is not compatible with HomeBridge Eufy Security plugin: Cam-1 Type: 63
[12/28/2023, 07:06:18]  [EufySecurity]  INFO    Looking for old cached accessories that seem to be outdated...
[12/28/2023, 07:06:18]  [EufySecurity]  INFO    No outdated cached accessories found.

### Device Type

Indoor Camera (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.6.x (Supported)

### NodeJS Version

v18 (Supported)

### Operating System

Ubuntu 18.04+ (Supported)

### Relevant log output

```shell
[2023-12-28, 7:05:33 a.m.] Homebridge v1.7.0 (HAP v0.11.1) (HB Hasen) is running on port 51259.
[12/28/2023, 07:05:39]  [EufySecurity]  INFO    Updating existing accessory: Cam-2
[12/28/2023, 07:05:39]  [EufySecurity]  INFO    Updating existing accessory: Cam-1
[12/28/2023, 07:05:41]  [EufySecurity]  WARN    This accessory is not compatible with HomeBridge Eufy Security plugin: Cam-2 Type: 63
[12/28/2023, 07:05:41]  [EufySecurity]  WARN    This accessory is not compatible with HomeBridge Eufy Security plugin: Cam-1 Type: 63
[12/28/2023, 07:06:18]  [EufySecurity]  INFO    Looking for old cached accessories that seem to be outdated...
[12/28/2023, 07:06:18]  [EufySecurity]  INFO    No outdated cached accessories found.
```

### config

```shell
{
            "username": "**********@gmail.com",
            "password": "********",
            "country": "US",
            "deviceName": "",
            "platform": "EufySecurity",
            "ignoreDevices": [
                "T8410P*****305DC"
            ],
            "cameras": [
                {
                    "serialNumber": "T813*****2003A9",
                    "rtsp": false,
                    "talkback": false
                },
                {
                    "serialNumber": "T81********001EA",
                    "rtsp": false,
                    "talkback": false
                }
            ],
            "ignoreStations": [
                "T**********305DC"
            ],
            "cleanCache": true
        }
```

**Comment by Infallibleman** [comment]
Yes,   Did a reset/clear cache on HB after the issue report.  Getting the following errors:

[12/28/2023, 09:16:46]  [EufySecurity]  ERROR   Cam-2 handleSecuritySystemCurrentStateGets: Wrong return value
[12/28/2023, 09:16:46]  [EufySecurity]  ERROR   Cam-2 handleSecuritySystemTargetStateGet: Wrong return value
[12/28/2023, 09:16:46]  [EufySecurity]  ERROR   Cam-1 handleSecuritySystemCurrentStateGets: Wrong return value
[12/28/2023, 09:16:46]  [EufySecurity]  ERROR   Cam-1 handleSecuritySystemTargetStateGet: Wrong return value

**Comment by jfegea3869** [comment]
same problem with s220

[01/27/2024, 11:04:22]  [EufySecurity]  ERROR   Terrasse handleSecuritySystemCurrentStateGets: Wrong return value
[01/27/2024, 11:04:22]  [EufySecurity]  ERROR   Terrasse handleSecuritySystemTargetStateGet: Wrong return value
[Terrasse 3ED2] The image snapshot handler for the given accessory is slow to respond! See https://homebridge.io/w/JtMGR for more info.
[01/27/2024, 11:04:41]  [EufySecurity]  ERROR   Error fetching snapshot: No snapshot in memory
[1/27/2024, 11:04:46 AM] [homebridge-eufy-security] This plugin slows down Homebridge. The write handler for the characteristic 'Selected RTP Stream Configuration' on the accessory 'Terrasse 3ED2' didn't respond at all!. Please check that you properly call the callback! See https://homebridge.io/w/JtMGR for more info.

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
